src/neuroconv/datainterfaces/text/timeintervalsinterface.py

```diff
@@ -42,6 +42,15 @@ def __init__(
         self.time_intervals = None
 
     def get_metadata(self) -> dict:
+        """
```

Comment:
goes against repeating some of those right?

Reply:
Yep, that's one of the main failures (I specifically told Cline not to add docstrings where they didn't already exist).

I tried to revert the extras, but looks like I missed some.